Adds documentation for exporting files (#1643)
webrecorder/browsertrix: docs/user-guide/archived-items.md

For more details on navigating web archives within ReplayWeb.page, see the [ReplayWeb.page user documentation.](https://replayweb.page/docs/exploring)

### Exporting Files

> While crawling, Browsertrix will output multiple WACZ files — the crawler aims to output files in consistently sized chunks, additionally each [crawler instance](workflow-setup.md#crawler-instances) will output separate WACZ files.

Reviewer: ```suggestion
While crawling, Browsertrix will output one or more WACZ files — the crawler aims to output files in consistently sized chunks, and each [crawler instance](workflow-setup.md#crawler-instances) will output separate WACZ files.
```

Author: Hell yeah consisce language changes!!
